opera-adt/COMPASS
# Some metadata fix for CSLC-S1

This PR contains some corrections/improvements to some metadata fields generated by the Cal/Val version of the CSLC-S1 SAS.

Some of the corrections stem from inherited and spurious NISAR metadata nomenclature. Others are done to ensure consistency through the entire generated product. 

TO DO:
1) Test the PR
2) Check if the CSLC-S1 product layers can still be imported in qGIS
3) Inspect the product again for other metadata inconsistencies
